framework/packages/abstract-core/src/app.rs

```diff
@@ -75,6 +78,17 @@ pub enum BaseQueryMsg {
     /// Returns the admin.
     #[returns(AdminResponse)]
     BaseAdmin {},
+    /// Returns module data
+    #[returns(AppModuleDataResponse)]
+    BaseModuleData {},
+}
+
+#[cosmwasm_schema::cw_serde]
+pub struct AppModuleDataResponse {
```

Comment:
`AppModuleDataResponse` and `AppModuleDataResponse` are the same and represent the same underlying structure (ModuleData). Can we instead add a single response type along with de definition of the ModuleData struct and return that in both implementation?